ivy/functional/ivy/general.py

```diff
@@ -556,31 +556,139 @@ def to_scalar(x: Union[ivy.Array, ivy.NativeArray]) -> Number:
 @inputs_to_native_arrays
 def to_list(x: Union[ivy.Array, ivy.NativeArray]) -> List:
     """Creates a (possibly nested) list from input array.
-
+    
     Parameters
     ----------
     x
         Input array.
-
+        
     Returns
     -------
     ret
         A list representation of the input array ``x``.
-
+        
     Examples
```

Comment:
Hi @husseinalamutu , thank you for making the edits!

Just a few more changes to make:

- [ ] 1. In line 570, please change `Examples` to `Functional Examples`.

- [ ] 2. Include a heading and section for `Instance Method Examples` for an ivy.Array instance method and an ivy.Container instance method.

- [ ] 3. Correct your usage of the ivy.Container input. (Have a look at my other comment for more detail.)